pkg/util/filesystem/permission_windows.go

```diff
@@ -0,0 +1,72 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+// +build windows
+
+package filesystem
+
+import (
+	"fmt"
+	"syscall"
+
+	"github.com/hectane/go-acl"
+	"golang.org/x/sys/windows"
+)
+
+// Handle permissions for Unix and Windows
+type Permission struct {
+	currentUserSid   *windows.SID
+	administratorSid *windows.SID
+	systemSid        *windows.SID
+}
+
+// Create a new instance of `Permission`
+func NewPermission() (*Permission, error) {
+	administratorSid, err := windows.StringToSid("S-1-5-32-544")
+	if err != nil {
+		return nil, err
+	}
+	systemSid, err := windows.StringToSid("S-1-5-18")
```

Comment:
I'm not familiar with windows: what are `S-1-5-32-544` and `S-1-5-18` ? Are they standard between all windows versions ?

Could we maybe have set in constant with a comment ? WDYT ?

Reply:
```go
administratorSid, err := windows.StringToSid("S-1-5-32-544")
```
As `administratorSid` suggest, `S-1-5-32-544` is the sid of administrator. Do you think extra comment is needed?

Reply:
I'd say if this is obvious to Windows dev we can leave it as is. If not, maybe a quick comment explaining it's an hardcoded ID for the Windows administrator and it's compatible with all windows version ?

You know more about Windows than me, I let you decide !